rules/expiring-todo-comments.js

```diff
@@ -284,6 +284,13 @@ const create = context => {
 
 	const {sourceCode} = context;
 	const comments = sourceCode.getAllComments();
+
+	const isEslintDirectiveComment = comment => {
+		// Strip leading whitespace and optional `*` from block comments
+		const normalizedComment = comment.value.trimStart().replace(/^\*\s*/, '');
+		return /^eslint(?:-(?:en|dis)able)?(?:-(?:next-)?line)?\b/.test(normalizedComment);
+	};
```

Comment:
Use `ConfigCommentParser` from `@eslint/plugin-kit` please.

You can take https://github.com/sindresorhus/eslint-plugin-unicorn/commit/0522c912869207b678cb7e9baf8ecdbfa1d1b451 as example